tools/loadout_converter/loadout_converter.json

```diff
@@ -320,6 +320,10 @@
 		"pants%2c+blue": "/datum/gear/uniform/pants/bluepants",
 		"trackpants": "/datum/gear/uniform/pants/trackpants",
 		"pants%2c+khaki": "/datum/gear/uniform/pants/khakipants",
+		"red+hawaiian+tshirt": "/datum/gear/uniform/suit/redhawaiianshirt",
+		"blue+hawaiian+tshirt": "/datum/gear/uniform/suit/bluehawaiianshirt",
+		"pink+hawaiian+tshirt": "/datum/gear/uniform/suit/pinkhawaiianshirt",
+		"orange+hawaiian+tshirt": "/datum/gear/uniform/suit/orangehawaiianshirt",
```

Comment:
This file was to convert pre-2021 loadouts to the new system, we probably don't need to maintain the file anymore? @hal9000PR please advise

Reply:
probably not

Reply:
sean is right, please remove this